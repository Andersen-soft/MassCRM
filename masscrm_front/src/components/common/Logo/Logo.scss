.icon {
  width: 31px;
  height: 31px;
}
