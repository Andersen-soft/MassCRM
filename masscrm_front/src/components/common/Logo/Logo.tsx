import React, { FC } from 'react';
import { styleNames } from 'src/services';
import styles from './Logo.scss';

const sn = styleNames(styles);

export const Logo: FC = () => {
  const icon = <use xlinkHref='assets/svg/sprite.svg#logo' />;

  return (
    <a href='/'>
      <svg className={sn('icon')}>{icon}</svg>
    </a>
  );
};
