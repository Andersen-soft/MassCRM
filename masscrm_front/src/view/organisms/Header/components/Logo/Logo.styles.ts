import { createStyles, makeStyles } from '@material-ui/core';

export const useStyles = makeStyles(() =>
  createStyles({
    icon: {
      width: '31px',
      height: '31px'
    }
  })
);
