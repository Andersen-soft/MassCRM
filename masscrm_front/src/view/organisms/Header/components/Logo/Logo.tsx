import React from 'react';
import { useStyles } from './Logo.styles';

export const Logo = () => {
  const icon = <use xlinkHref='assets/svg/sprite.svg#logo' />;
  const styles = useStyles();

  return (
    <a href='/'>
      <svg className={styles.icon}>{icon}</svg>
    </a>
  );
};
