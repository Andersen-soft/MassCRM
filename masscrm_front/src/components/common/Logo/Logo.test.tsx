import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { Logo } from './Logo';

describe('Logo', () => {
  it('Logo should render correctly', () => {
    expect(shallowToJson(shallow(<Logo />))).toMatchSnapshot();
  });
});
