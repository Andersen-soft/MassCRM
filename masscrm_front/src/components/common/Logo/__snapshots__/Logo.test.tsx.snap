// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Logo should render correctly 1`] = `
<a
  href="/"
>
  <svg
    className="icon"
  >
    <use
      xlinkHref="assets/svg/sprite.svg#logo"
    />
  </svg>
</a>
`;
